tasks/components/cards/PatientCard.tsx

```diff
@@ -27,8 +27,8 @@ export const PatientCard = ({
   return (
     <Card isSelected={isSelected} onTileClick={onTileClick}>
       <div className={tw('flex flex-row justify-between')}>
-        <Span type="subsubsectionTitle">{bedName}</Span>
-        <Span>{patientName}</Span>
+        <Span className={tw('whitespace-nowrap')} type="subsubsectionTitle">{bedName}</Span>
+        <span className={tw('ml-2 text-ellipsis overflow-hidden block  whitespace-nowrap')}>{patientName}</span>
```

Comment:
preferably use truncate

Reply:
Instead of text-ellipsis?

Reply:
truncate does this in tailwind and doesn't require you to write four key words, and also use our span for easier typography management. The line below does exactly what you want
```suggestion
        <Span className={tw('ml-2 truncate')}>{patientName}</Span>
```